sno/init.py

```diff
@@ -245,6 +268,18 @@ def import_table(
             description=info.get("description"),
             xml_metadata=info.get("xmlMetadata"),
         )
+        if replace_ids is not None:
```

Comment:
seems like this is only valid if you specify `--replace-existing` too?

Reply:
it implies it. i'll add to help text